### PR TITLE
Adds Grape Jelly and missing Condiment Bottles

### PR DESCRIPTION
**Pull request by Roostercat12**
Adds Grape Jelly as a reagent obtainable by blending grapes with its own condiment bottle sprite to boot. Grape recipes to come later. 

In addition, adds condiment bottle sprites for the following ingredients: Ambrosia/Wulu extract, Psilocybin, Batter, Pumpkin Pulp, Pumpkin Spice, Cocoa Powder, Egg Yolk, Triglyceride.

All sprites made by myself. 

sick condiment bottles

### CODE REVIEW

**Comment by Roostercat12** [comment]
!review

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 1]
```diff
@@ -319,6 +331,15 @@
 
 	return min((holder.get_temperature() - threshold)/step, 2.5)
 
+/decl/reagent/nutriment/grapejelly
+	name = "Grape Jelly"
+	description = "A jelly produced from blending grapes."
+	color = "#410083"
+	taste_description = "grapes"
+	condiment_name = "Grape Jelly"
+	condiment_desc = "A jar of jelly derived from grapes. Superior to Cherry Jelly."
```
Comment (SleepyGemmy) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:340]:
```suggestion
	condiment_desc = "A jar of jelly derived from grapes. Superior to cherry jelly."
```

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 2]
```diff
@@ -4909,13 +4934,21 @@
 	description = "The gooey insides of a slain pumpkin. This day is the greatest..."
 	color = "#f9ab28"
 	taste_description = "gooey pumpkin"
+	condiment_name = "Pumpkin Pulp Jar"
+	condiment_desc = "An orange jar with a picture of a Pumpkin on its label. Spooky."
```
Comment (SleepyGemmy) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:4938]:
```suggestion
	condiment_desc = "An orange jar with a picture of a pumpkin on its label. Spooky."
```

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 3]
```diff
@@ -4909,13 +4934,21 @@
...
+	condiment_center_of_mass = list("x"=16, "y"=8)
 
 /decl/reagent/spacespice/pumpkinspice
 	name = "Pumpkin Spice"
 	description = "A delicious seasonal flavoring."
 	color = "#AE771C"
 	taste_description = "autumn bliss"
 	condiment_name = "bottle of pumpkin spice"
+	condiment_name = "Pumpkin Spice"
+	condiment_desc = "Every teenage girl's favorite seasonal ingredient."
```
Comment (SleepyGemmy) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:4949]:
```suggestion
	condiment_desc = "Every teenager's favorite seasonal ingredient."
```

File: html/changelogs/Grape Jelly And Condiment Bottles.yml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.
+author: ChangeMe
```
Comment (SleepyGemmy) [inline html/changelogs/Grape Jelly And Condiment Bottles.yml:30]: faulty author.

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

File: html/changelogs/Grape Jelly And Condiment Bottles.yml (hunk 4 above)
Comment (Roostercat12) [reply html/changelogs/Grape Jelly And Condiment Bottles.yml]: fixed

**Review by SleepyGemmy — APPROVED** [review]
suggested changes addressed.

**Review by KingOfThePing — APPROVED** [review]
Another PR Gem cheats himself into being a co-author, god bless

**Review by Alberyk — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**